app/db_models.py

```diff
@@ -345,6 +347,64 @@ def update_obj(self, question, data):
             if field in data:
                 setattr(question, field, data[field])
 
+class FillInTheBlankQuestion(Question):
+    id = db.Column(db.Integer, db.ForeignKey('question.id'), primary_key=True)
+
+    question_text = db.Column(db.String, nullable=False)
```

Comment:
The base `Question` class has a prompt Column, which means we shouldn't need this `question_text` column.